client/src/app/site/projector/components/countdown-controls/countdown-controls.component.ts

```diff
@@ -44,6 +45,11 @@ export class CountdownControlsComponent extends BaseViewComponent {
      */
     public warningTime: number;
 
+    /**
+     * The key to storage.
+     */
+    private storageKey = 'projectorElementOptions';
```

Comment:
The value `projectorElementOptions` is used twice in this PR. Please put it in a constant above the class, export it and reuse the constant.